kuttl-test-upgrade.yaml

```diff
@@ -14,13 +14,12 @@ kind: TestSuite
 crdDir: ./tests/_build/crds/
 artifactsDir: ./tests/_build/artifacts/
 kindContainers:
-  - local/opentelemetry-operator:e2e
   - ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:v0.49.0
 commands:
-  - command: make cert-manager
```

Comment:
note that there should be be dependencies between tests. 

The test should be possible to run without assumptions on running any previous tests.

Reply:
Yes, I know. But the `cert-manager` will be deployed during `prepare-e2e`. It'll be not modified by the tests.. so, it is not needed to deploy it again, right?